build/ableplayer.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'ableplayer';
```

Comment:
I don't *have* to check it in, but so far all generated files are checked into the `build` dir, as per the original repo